### DISCUSSION TITLE
Per-User default shell?

### DISCUSSION THREAD

**Discussion by Khitiara** [PowerShell/Win32-OpenSSH#2191] [Q&A] [answered]
Id like to set a per-user default shell as i have powershell installed from the windows store on the computer i am SSH-ing into and would like to set that as the default shell for just my user - i see the docs for the main default shell option but this seems to be a whole-machine option rather than per-user

**Comment by maertendMSFT** [comment, marked as answer]
We read this information from the Windows registry and it is not a setting that is explicitly controlled by SSH. We believe that it would be possible to support per user default shells, and we have added an issue to track the feature request: https://github.com/PowerShell/Win32-OpenSSH/issues/2198
